oauth2client/contrib/gce.py

```diff
@@ -135,6 +136,8 @@ def _refresh(self, http_request):
                 raise HttpAccessTokenRefreshError(str(e),
                                                   status=response.status)
             self.access_token = token_content['access_token']
+            delta = datetime.timedelta(seconds=int(token_content['expires_in']))
```

Comment:
Oh. Is the 0.1 metadata service more recent than v1? Or a different API? I don't have it in front of me but IIRC the v1 server only returned the seconds until expiration.